src/components/Nav/Nav.js

```diff
@@ -13,14 +13,14 @@ const Nav = () => {
             </Link>
           </li>
           {MAIN_MENU_LIST.map(info => (
-            <li className="menulist" key={info.key}>
+            <li className="menulist" key={info.id}>
```

Comment:
같은 파일 23번째 줄에서도 map을 활용하고 있고 둘다 상수데이터를 활용하고있기때문에 아무래도 key값이 중복될 수도 있을것 같습니다 이럴때는 key값에 prefix를 붙여 좀 더 유니크한 값을 부여할 수 있습니다
```suggestion
            <li className="menulist" key={`menu-list-${info.id}`}>
```